src/MDMPerforming.h

```diff
@@ -148,6 +148,30 @@ NS_SWIFT_NAME(ComposablePerforming)
 
 @end
 
+/** Specifics for a named plan performer to allow named plans to be added and removed. */
+NS_SWIFT_NAME(NamedPlanPerforming)
+@protocol MDMNamedPlanPerforming <MDMPerforming>
+
+/**
+ Provides the performer with a plan and an associated name.
+
+ @param plan The plan that required this type of performer.
+ @param name The name by which the plan can be identified.
+ */
+- (void)addPlan:(nonnull id<MDMPlan>)plan
+          named:(nonnull NSString *)name
+  NS_SWIFT_NAME(addPlan(plan:named:));
```

Comment:
Ah - I was futzing around with `NS_SWIFT_NAME` forever trying to figure out how to have an unnamed param. Thanks for the pointer.